packages/remix-dev/vite/plugin.ts

```diff
@@ -197,7 +197,6 @@ export type ResolvedVitePluginConfig = Pick<
   buildDirectory: string;
   manifest: boolean;
   serverBuildFile: string;
-  serverBundleId?: string;
```

Comment:
This is the core change in this PR. Everything else is in service of this change.